src/app/desktop/windows/terminal/terminal-states.ts

```diff
@@ -907,21 +1017,21 @@ export class DefaultTerminalState extends CommandTerminalState {
             service_uuid: bruteforceService['uuid'], device_uuid: activeDevice,
             target_device: targetDevice, target_service: targetService
           }).subscribe(() => {
-              this.terminal.outputText('You started a bruteforce attack');
-              this.terminal.pushState(new BruteforceTerminalState(this.terminal, this.domSanitizer, stop => {
-                if (stop) {
-                  this.executeCommand('service', ['bruteforce', targetDevice, targetService]);
-                }
-              }));
-            }, error1 => {
-              if (error1.message === 'could_not_start_service') {
-                this.terminal.outputText('There was an error while starting the bruteforce attack');
-              } else if (error1.message === 'invalid_input_data') {
-                this.terminal.outputText('The specified UUID is not valid');
-              } else {
-                reportError(error1);
+            this.terminal.outputText('You started a bruteforce attack');
+            this.terminal.pushState(new BruteforceTerminalState(this.terminal, this.domSanitizer, stop => {
+              if (stop) {
+                this.executeCommand('service', ['bruteforce', targetDevice, targetService]);
               }
+            }));
+          }, error1 => {
+            if (error1.message === 'could_not_start_service') {
+              this.terminal.outputText('There was an error while starting the bruteforce attack');
+            } else if (error1.message === 'invalid_input_data') {
+              this.terminal.outputText('The specified UUID is not valid');
+            } else {
+              reportError(error1);
             }
+          }
```

Comment:
Don't change the styling

Reply:
Wie kann ich in VS Code das Automatisch Formatieren so einstellen das so etwas nicht passsiert